src/pages/blog/2020-11-19-best-papers.md

```diff
@@ -0,0 +1,58 @@
+---
+templateKey: blog-post-page
+title: "Best Paper Awards"
+date: 2020-11-19
+seo:
+  title: "Best Paper Awards for EMNLP 2020"
+  description: >- 
+    Here we announce the award winning papers for EMNLP, including honorable mentions and best demonstration paper. 
+---
+
+The Best Paper Awards recognise many of the outstanding papers appearing at
+EMNLP 2020. We considered for these awards both long and short papers, as well
+as demonstration papers.
+
+The main paper awards were selected as follows:
+ - Based on paper reviews, the opinions of the area chair, and the senior area
+   chair(s), we constructed a short-list of strong papers. We ensured
+   that the short-list contained papers of a diversity of different types:
+   e.g., resources papers, theory papers, papers from most of the conference tracks, 
+   and short and long format papers.
+ - The short-listed papers were sent to a [Best Paper committee](https://2020.emnlp.org/organizers/best-paper-committee) 
+   for further review, from which the overall top papers were awarded honourable mentions
+   and best paper.
+    
+Note that the best demonstration paper was decided differently, being chosen directly by the Demonstration chairs.
+
+The following papers have been given awards.
+
+**Best Paper**
+
+Digital voicing of Silent Speech  
+*David Gaddy and Dan Klein*
+
+**Honourable Mention Papers**
+
+If beam search is the answer, what was the question?  
+*Clara Meister, Ryan Cotterell and Tim Vieira*
+
+GLUCOSE: GeneraLized and COntextualized Story Explanations  
+*Nasrin Mostafazadeh, Aditya Kalyanpur, Lori Moon, David Buchanan, Lauren Berkowitz, Or Biran and Jennifer Chu-Carroll*
+
+Spot The Bot: A Robust and Efficient Framework for the Evaluation of Conversational Dialogue Systems  
+*Jan Deriu, Don Tuggener, Pius von Däniken, Jon Ander Campos, Alvaro Rodrigo, Thiziri Belkacem, Aitor Soroa, Eneko Agirre and Mark Cieliebak*
+
+Visually Grounded Compound PCFGs  
+*Yanpeng Zhao and Ivan Titov*
+
+**Best Short Paper**
+
+Not awarded.
+
+**Best Demonstration Paper**
+
+:hugging_face: Transformers: State-of-the-art Natural Language Processing  
```

Comment:
is this `:hugging_face:` supposed to be an emoji or this literal string @trevorcohn ?

Reply:
Emoji, ideally. But either works -- as a string it would be capitalised HuggingFace.